docs/inference-providers/guides/image-editor.md

```diff
@@ -0,0 +1,324 @@
+)
+
+def edit_image(input_image, prompt):
+    """
+    Edit an image using the given prompt.
+    
+    Args:
+        input_image: PIL Image object from Gradio
+        prompt: String prompt for image editing
+    
+    Returns:
+        PIL Image object (edited image)
+    """
+    if input_image is None:
+        return None
+    
+    if not prompt or prompt.strip() == "":
+        return input_image
+    
+    try:
+        # Convert PIL Image to bytes
+        img_bytes = io.BytesIO()
+        input_image.save(img_bytes, format="PNG")
+        img_bytes = img_bytes.getvalue()
+        
+        # Use the image_to_image method
+        edited_image = client.image_to_image(
+            img_bytes,
+            prompt=prompt.strip(),
+            model="Qwen/Qwen-Image-Edit",
+        )
+        
+        return edited_image
+    
+    except Exception as e:
+        print(f"Error editing image: {e}")
+        return input_image
+
+# Create Gradio interface
+with gr.Blocks(title="Image Editor", theme=gr.themes.Soft()) as interface:
+    gr.Markdown(
+        """
+        # 🎨 AI Image Editor
+        Upload an image and describe how you want to edit it using natural language!
+        """
+    )
+
+    with gr.Row():
+        with gr.Column():
+            input_image = gr.Image(label="Upload Image", type="pil", height=400)
+            prompt = gr.Textbox(
+                label="Edit Prompt",
+                placeholder="Describe how you want to edit the image...",
+                lines=2,
+            )
+            edit_btn = gr.Button("✨ Edit Image", variant="primary", size="lg")
+
+        with gr.Column():
+            output_image = gr.Image(label="Edited Image", type="pil", height=400)
+
+    # Example images and prompts
+    with gr.Row():
+        gr.Examples(
+            examples=[
+                ["cat.png", "Turn the cat into a tiger"],
+                ["cat.png", "Make it look like a watercolor painting"],
+                ["cat.png", "Change the background to a forest"],
+            ],
+            inputs=[input_image, prompt],
+            outputs=output_image,
+            fn=edit_image,
+            cache_examples=False,
+        )
+
+    # Event handlers
+    edit_btn.click(fn=edit_image, inputs=[input_image, prompt], outputs=output_image)
+
+    # Allow Enter key to trigger editing
+    prompt.submit(fn=edit_image, inputs=[input_image, prompt], outputs=output_image)
+
+if __name__ == "__main__":
+    interface.launch(
+        share=True,  # Creates a public link
+        server_name="0.0.0.0",  # Allow external access
+        server_port=7860,  # Default Gradio port
+        show_error=True,  # Show errors in the interface
+    )
+```
+
+</details>
+
+## Deploy on Hugging Face Spaces
+
+1. **Create a new Space**: Go to [huggingface.co/new-space](https://huggingface.co/new-space)
+2. **Choose Gradio SDK** and make it public
+3. **Upload your files**: Upload `main.py` and any example images
```

Comment:
Also `requirements.txt` if necessary, no? Users would have to create it manually from the toml file (unless there's a way to automate it)